config/default.py

```diff
@@ -102,8 +102,16 @@
 EMAIL_SYSTEM_ERRORS_TO = ['__NESSIE_at_berkeley.edu']
 EMAIL_TEST_MODE = True
 
+# Broad-based EDL feature flag.
 FEATURE_FLAG_ENTERPRISE_DATA_LAKE = False
 
+# And more granular EDL feature flags.
+FEATURE_FLAG_EDL_DEGREE_PROGRESS = False
+FEATURE_FLAG_EDL_DEMOGRAPHICS = False
+FEATURE_FLAG_EDL_REGISTRATIONS = False
+FEATURE_FLAG_EDL_SIS_VIEWS = False
+FEATURE_FLAG_EDL_STUDENT_PROFILES = False
```

Comment:
@lyttam @johncrossman Note these more granular feature flags which I'm hoping will help organize us going forward. Everything we're doing this sprint should fall under `FEATURE_FLAG_EDL_SIS_VIEWS`.